ast/src/circuits/circuit_member.rs

```diff
@@ -21,11 +21,17 @@ use std::fmt;
 
 #[derive(Clone, PartialEq, Eq, Serialize, Deserialize)]
 pub enum CircuitMember {
-    // (const_name, const_type, const_expression)
+    /// Represented as three elements.
+    /// const_name - An Identifier representing the constant name.
+    /// const_type - The type of the constant.
+    /// const_expression - The expression representing the value of the constant.
```

Comment:
I was more thinking documenting what this means in syntax, so that a reader knows how to map this from Leo source code to the AST. See for example https://doc.rust-lang.org/nightly/nightly-rustc/rustc_ast/ast/enum.ExprKind.html. "Represented as 3 elements" doesn't add anything really as I can see that from the Rust code here. Also, documenting the fields can be done directly via doc comments on those.

Reply:
Aren't those comments precisely what it means in syntax? You have a name, type, and expression. At the very least, I'm not sure of another way to describe it. Also, those comments are above each respective enum member.

Reply:
Something like this:
```rust
/// A static constant in a circuit.
/// For example: `const foobar: u8 = 42;`.
CircuitConst(
    /// The identifier of the constant.
    Identifier,
    /// The type the constant has.
    Type,
    /// The expression representing the constant's value.
    /// Checked to be of the type above.
    Expression
),
```